src/terrama2/services/view/core/data-access/Geoserver.cpp

```diff
@@ -926,24 +926,49 @@ void terrama2::services::view::core::GeoServer::registerMosaicCoverage(const std
                                        + storeName
                                        + "/coverages/" + coverageName);
 
-    std::string xml = "<coverage>"
-                      "<enabled>true</enabled>"
-                      "<srs>EPSG:"
-                      + std::to_string(srid) +
-                      "</srs>"
-                      "<metadata>"
-                      "<entry key=\"time\">"
-                      "<dimensionInfo>"
-                      "<enabled>true</enabled>"
-                      "<presentation>LIST</presentation>"
-                      "<units>ISO8601</units>"
-                      "<defaultValue>"
-                      "<strategy>MAXIMUM</strategy>"
-                      "</defaultValue>"
-                      "</dimensionInfo>"
-                      "</entry>"
-                      "</metadata>"
-                      "</coverage>";
+    std::string xml = "<coverage>";
+
+    // Using TerraMA² toString to keep precision
+    const std::string llx = toString(rasterInfo.envelope->getLowerLeftX(), 14);
+    const std::string urx = toString(rasterInfo.envelope->getUpperRightX(), 14);
+    const std::string lly = toString(rasterInfo.envelope->getLowerLeftY(), 14);
+    const std::string ury = toString(rasterInfo.envelope->getUpperRightY(), 14);
+    const std::string srid = std::to_string(rasterInfo.srid);
+    // Settings bounds
+    xml += "<nativeBoundingBox>"
+           "  <minx>"+llx+"</minx>"
+           "  <maxx>"+urx+"</maxx>"
+           "  <miny>"+lly+"</miny>"
+           "  <maxy>"+ury+"</maxy>"
+           "  <crs>EPSG:"+srid+"</crs>"
+           "</nativeBoundingBox>";
+
+    // Settingd grid
+    xml += "<latLonBoundingBox>"
+           "  <minx>"+llx+"</minx>"
+           "  <maxx>"+urx+"</maxx>"
+           "  <miny>"+lly+"</miny>"
+           "  <maxy>"+ury+"</maxy>"
+           "  <crs>EPSG:"+srid+"</crs>"
+           "</latLonBoundingBox>";
```

Comment:
The latLonBoundingBox should alway be in EPSG:4326,
we must convert the coordinates before setting.